code/game/gamemodes/wizard/rightandwrong.dm

```diff
@@ -49,7 +49,7 @@
 					new /obj/item/weapon/gun/projectile/pistol(get_turf(H))
 					new /obj/item/gun_part/silencer(get_turf(H))
 				if("cannon")
-					new /obj/item/weapon/gun/energy/lasercannon(get_turf(H))
+					new /obj/item/weapon/gun/energy/laser/cannon(get_turf(H))
```

Comment:
Or maybe it should have fully explained what the PR did, because you're insane to think everyone has nothing better to do than go into GitHub and sift through the smokescreen of "it's not that big if a nerf, I swear" garbage in the first post. 

The poll simply does not reflect the actual feelings of the server, and if you try to force it through now you will find that out the hard way.